lib/ansible/plugins/action/include_vars.py

```diff
@@ -40,12 +40,14 @@ def _set_dir_defaults(self):
 
         if isinstance(self.ignore_files, string_types):
```

Comment:
Use `str` here, so the import for `string_types` can be removed:

```suggestion
        if isinstance(self.ignore_files, str):
```